src/redux/reducers/MyUpNextReducer.js

```diff
@@ -0,0 +1,35 @@
+import {
+  FETCH_MY_UP_NEXT_START,
+  FETCH_MY_UP_NEXT_SUCCESS,
+  FETCH_MY_UP_NEXT_FAILURE,
+  TOGGLE_MY_UP_NEXT
+} from '../actions/myUpNextActions';
+
+const toggleUpNextItem = (state, action) => {
+  let newItems = state.items.map((item, index) => {
+    // set as false if not the toggled item
+    if (index !== action.index) {
+      return { ...item, show: false }
+    }
+    // reverse the expanded state of the item clicked on
+    return { ...item, show: !item.show }
+  });
+  return { ...state, items: newItems };
+};
```

Comment:
If I understand this correctly, only one up next item is shown at a time, right? Again this could be in Redux or React useState. If you want to discuss, cool but I'm fine with this.

Reply:
@developish - Yes. Let's discuss because I'm not sure how I could have done that with `useState`.

Reply:
Discussed with Brandon and he recommended using a single value in the UpNextCard component that represents the index value of the component that should be expanded.